build-docker-images.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This should probably be added to the README.md at some point.

Reply:
I'm thinking to add it to the `DEVELOPMENT.md` in future.